FlightJournal.Web/Translations/LocalizedWebViewPage.cs

```diff
@@ -16,7 +16,14 @@ public abstract class LocalizedWebViewPage<TModel> : WebViewPage<TModel>
         /// <example>@_("Country")</example>
         public string _(string en)
         {
-            return Internationalization.GetText(en, Internationalization.LanguageCode);
+            var s = Internationalization.GetText(en, Internationalization.LanguageCode);
+            if (s != " ")
+                return s.Trim();
+            return s;
+        }
+        public string __(string en)
```

Comment:
- [does this get picked up by po parser]

Reply:
It does, nice work. we should consider comments tho

Reply:
That actually took more or less a full day to get right, having considered all sorts of script-based hacks to get usage in strings to work.

The simplest would have been to grep for "_(" and let xgettext/po work on the output, but that would drop all real file references (and windows doesn't have grep...)